recipe-hooks-project/src/Register.js

```diff
@@ -26,15 +26,11 @@ function Register() {
     }
   }
 
-  const renderRedirect = () => {
-    if (redirect) {
-      return <Redirect to="/app"/>;
-    }
-  }
-
   return (
     <>
-      {renderRedirect()}
+      {
```

Comment:
It's a bit uncommon to leave a redirect inside the body, I would just put a separate `if` before this `return`.
```tsx
if (redirect) return <Redirect to="/app" />

return (
  <>
    <Container />
  </>
```